azurerm/internal/services/compute/virtual_machine_scale_set.go

```diff
@@ -779,6 +779,18 @@ func VirtualMachineScaleSetDataDiskSchema() *schema.Schema {
 					Optional: true,
 					Default:  false,
 				},
+
+				"disk_iops_read_write": {
+					Type:     schema.TypeInt,
+					Optional: true,
+					Computed: true,
+				},
+
+				"disk_mbps_read_write": {
+					Type:     schema.TypeInt,
+					Optional: true,
+					Computed: true,
```

Comment:
what are these set to when the disk is not an ultra ssd?

Reply:
In this case, the service does not return a value for this. In the terraform state, this will be set to 0